docs/book/v6/tutorials/create-book-module.md

```diff
@@ -7,65 +7,102 @@ The below files structure is what we will have at the end of this tutorial and i
+    │       │   └── Book.php
+    │       ├── Handler/
+    │       │   ├── GetBookCollectionHandler.php
+    │       │   ├── GetBookResourceHandler.php
+    │       │   └── PostBookResourceHandler.php
+    │       ├── InputFilter/
+    │       │   ├── Input/
+    │       │   │   ├── AuthorInput.php
+    │       │   │   ├── NameInput.php
+    │       │   │   └── ReleaseDateInput.php
+    │       │   └── CreateBookInputFilter.php
+    │       ├── Service/
+    │       │   ├── BookService.php
+    │       │   └── BookServiceInterface.php
+    │       ├── ConfigProvider.php
+    │       └── RoutesDelegator.php
+    └── Core/
         └── src/
-            ├── Collection/
-            │   └── BookCollection.php
-            ├── Entity/
-            │   └── Book.php
-            ├── Handler/
-            │   └── BookHandler.php
-            ├── InputFilter/
-            │   ├── Input/
-            │   │   ├── AuthorInput.php
-            │   │   ├── NameInput.php
-            │   │   └── ReleaseDateInput.php
-            │   └── BookInputFilter.php
-            ├── Repository/
-            │   └── BookRepository.php
-            ├── Service/
-            │   ├── BookService.php
-            │   └── BookServiceInterface.php
-            ├── ConfigProvider.php
-            └── RoutesDelegator.php
+            └── Book/
+                └── src/
+                    ├──Entity/
+                    │   └──Book.php
+                    ├──Repository/
+                    │   └──BookRepository.php
+                    └── ConfigProvider.php
+    
 ```
 
-* `src/Book/src/Collection/BookCollection.php` - a collection refers to a container for a group of related objects, typically used to manage sets of related entities fetched from a database
-* `src/Book/src/Entity/Book.php` - an entity refers to a PHP class that represents a persistent object or data structure
-* `src/Book/src/Handler/BookHandler.php` - handlers are middleware that can handle requests based on an action
-* `src/Book/src/Repository/BookRepository.php` - a repository is a class responsible for querying and retrieving entities from the database
-* `src/Book/src/Service/BookService.php` - is a class or component responsible for performing a specific task or providing functionality to other parts of the application
-* `src/Book/src/ConfigProvider.php` -  is a class that provides configuration for various aspects of the framework or application
-* `src/Book/src/RoutesDelegator.php` - a routes delegator is a delegator factory responsible for configuring routing middleware based on routing configuration provided by the application
-* `src/Book/src/InputFilter/BookInputFilter.php` - input filters and validators
-* `src/Book/src/InputFilter/Input/*` - input filters and validator configurations
+* `src/Book/src/Collection/BookCollection.php` – a collection refers to a container for a group of related objects, typically used to manage sets of related entities fetched from a database
```

Comment:
By

> Also, let's sort these items.

and

> ... because that's how they appear in the IDE

I meant: sort these items both alphabetically AND hierarchically (maybe I should have made this clearer).
That meand first, by directory alphbetically and inside each directory the files are also sorted alphabetically - like you did in the above tree.